plugins/net-forward.yaml

```diff
@@ -23,12 +23,17 @@ spec:
     uri: https://github.com/antitree/krew-net-forward/releases/download/v1.0.3/net-forward_v1.0.3.tar.gz
     sha256: "645adb05d9ad68ecc91d9034cb80b1e2fe4762864323c12de0f2bc5afe20baa0"
     bin: "net-forward"  
-  shortDescription: "Setup a true forwarder from your local machine to any TCP service accessible from within the cluster."
+  shortDescription: "Extend the functionality of port-forward connect to any IP and port (not just Pods and Services)."
```

Comment:
When listing plugins, the `shortDescription` get truncated at 53 chars. Anything longer than that is usually worthless.

Reply:
shortened to 53 chars